### DISCUSSION TITLE
Twinkle Tray needs DCC/CI Support??? What does that mean?

### DISCUSSION THREAD

**Discussion by 4RealHaxt0rZ** [xanderfrangos/twinkle-tray#853] [General]
Not sure where I am supposed to ask for help but I cannot use Twinkle Tray because I need DCC/CI? I have no clue what that means, I have always used PangoBright since my computer does not have brightness functionality for some reason, My computer is connected through HDMI to a Samsung TV if that means anything, If I can't get DCC/CI I might just have to stick with PangoBright which sucks because it disables my ability to hide my taskbar which I need for windowed borderless gaming

**Comment by PowerPCFan** [comment]
DDC/CI is the technology that Twinkle Tray uses to communicate with monitors to change brightness and other settings, I don't think a TV has that so it probably won't work in your scenario.
